src/Impl/GuzzleHttpRequester.php

```diff
@@ -85,8 +94,12 @@ private function parseMessageStatus(ResponseInterface $response): MessageStatus
     private function parseFailureResponse(ResponseInterface $response): FailureResponse
     {
         try {
-            $jsonObject = $this->parseResponseToJson($response);
-            return FailureResponse::fromJson($jsonObject);
+            $jsonObjects = $this->parseResponseToJsons($response);
+            $jsonObjectsCount = count($jsonObjects);
+            if ($jsonObjectsCount !== 1) {
```

Comment:
There may be many reasons.

Reply:
According to the current implementation of the Ingestion Service, there should be
 - either a single "severe" failure
 - or a list to single successes/failures

depending on the Http Status